Streams/Streams.tex

```diff
@@ -313,8 +313,7 @@ \subsection{Writing to collections}
 
 \paragraph{About Concatenation.}
 Using \mthind{WriteStream}{nextPut:} and \mthind{WriteStream}{nextPutAll:} on a \ct{WriteStream} is often the best way to concatenate characters.
-Using the comma concatenation operator (\ct{,}) is far less efficient:
-\index{Collection!comma operator}

```

Comment:
I like the index entry, but maybe with concatenation operator instead of comma operator?

Reply:
Is "operator" really Smalltalk vocabulary? Neither the compiler/parser/scanner nor Symbol in the image do use it. On the other hand, I realize the book already makes heavy use of it. So I will restore it ...

Reply:
@LinqLover true, maybe use "Collection!concatenation"?

Reply:
Better? https://github.com/hpi-swa-lab/SqueakByExample-english/pull/179/files/6db46707bf6266b5bccad98eeba0d03462b90cbb..2ded76bc476a94c670ec0b533053d966362a68e7